distribution/kubernetes/vector-stateless-aggregator/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
This looks like a valid URL, but doesn't resolve. Is that intended or something to worry about?

Reply:
It's the scheme kubernetes uses: [see versioning policy](https://kubectl.docs.kubernetes.io/faq/kustomize/versioningpolicy/#default-values-for-k8s-kind-and-apiversion

It's in the form of `kind/version`